node.js

```diff
@@ -49,7 +49,42 @@ module.exports = {
     'space-before-function-paren': off,
     'keyword-spacing': error,
     'eol-last': [ error, always ],
-    'brace-style': [ error, '1tbs', { allowSingleLine: true } ],
-    'curly': [ error, all ]
+    'brace-style': [ error, '1tbs', { allowSingleLine: false } ],
+    'curly': [ error, all ],
+    'block-spacing': [ error,  always ],
+    'indent': [ error, 2, { SwitchCase: 1 } ],
+    'no-tabs': error,
+    'spaced-comment': [ error, always ],
+    'array-callback-return': warn,
+    'eqeqeq': [ error, always, { null: 'ignore' } ],
+    'no-else-return': [ warn, { allowElseIf: false } ],
+    'yoda': error,
+    'no-throw-literal': error,
+    'no-self-compare': error,
+    'no-return-assign': error,
+    'no-loop-func': error,
+    'valid-jsdoc': warn,
+    'for-direction': error,
+    'no-buffer-constructor': error,
+    'no-new-require': error,
+    'no-path-concat': error,
+    'comma-spacing': error,
+    'comma-style': error,
+    'computed-property-spacing': [ error, never ],
+    'consistent-this': [ error, 'self' ],
+    'func-call-spacing': [ error, never ],
+    'key-spacing': error,
+    'new-cap': error,
+    'no-lonely-if': error,
+    'no-mixed-operators': error,
+    'no-unneeded-ternary': [ error, { defaultAssignment: false } ],
+    'quote-props': [ warn, 'consistent-as-needed' ],
+    'space-before-blocks': error,
+    'space-in-parens': [ error, never ],
+    'space-unary-ops': error,
+    'space-infix-ops': error,
+    'prefer-const': warn,
```

Comment:
Prefer usage of constants when a variable is never reassigned
https://eslint.org/docs/rules/prefer-const
```js
// Old
let a = 1
for (let prop in obj) {
  // ...
}

// New
const a = 1
for (const prop in obj) {
  // ...
}

Reply:
agree but this will be annoying when prototyping

Reply:
also `for` is not really the example I usually face up

```js
let val = valTwo + "something"
const valThree = valTwo + "something else"
``` 

when you prefer it instead of:

```
let val = valTwo + "something"
const valThree = valTwo + "something else"
```

ok you could ignore the error, but not before checking if this was really an linting error or a false-positive (when prototyping)